lusidtools/cocoon/properties.py

```diff
@@ -250,6 +249,25 @@ def create_missing_property_definitions_from_file(
     # If there are property columns
     if len(property_columns) > 0 and domain is not None:
```

Comment:
Update type hint of property columns to be list[dict]

Reply:
Had a problem with some dynamic code when I did this. Have pasted the stack trace above

Reply:
Okay fair enough.